src/theme/CodeTabs.js

```diff
@@ -11,6 +11,7 @@ export default function CodeTabs({ children }) {
             { label: 'JavaScript', value: 'js', },
             { label: 'Python', value: 'py', },
             { label: 'Go', value: 'go', },
+            ...additionalTabs,
```

Comment:
This is temporary until we get the ruby & php libraries to parity

Reply:
Yeah, sounds good!